### ISSUE TITLE
Need "Registration" page.

### ISSUE THREAD

**Issue by utkarsh2102** [GlobalTechTalks/hackathon#8]
We need a `Registration` page for the website.
The common details like Name, Email-ID, Contact Number, GitHub and Twitter handle and a reCAPTCHA are needed.
The UI should be as good as possible.
Contribution in any kind is accepted :D
NOTE: Work on a separate branch.
Filename could be `registration.html`.

**Comment by kartikeytripathi** [comment]
Here I am 😉

**Comment by utkarsh2102** [comment]
Thanks, @kartikeytripathi.
Please raise a PR at the earliest on the `develop` branch ;)

**Comment by GouravSardana** [comment]
@utkarsh2102 Leave the captcha part.. I will add it from Django.

**Comment by utkarsh2102** [comment]
Would that be easy enough, @GouravSardana?

**Comment by GouravSardana** [comment]
@utkarsh2102  Yup... you just only want to use a jquery prevent for the submit button i.e it would be submit until you will get response from the captcha.
